templates/page_not_found.html

```diff
@@ -0,0 +1,5 @@
+{% extends "layout.html" %}
+
+{% block body %}
+Item not found fuck off!
```

Comment:
watch you language 
![](https://media.giphy.com/media/l4FGtq6PhsePqpMlO/giphy.gif)